crates/db-boot/src/x86_64/mod.rs

```diff
@@ -3,8 +3,9 @@
 
 //! VM boot related constants and utilities for `x86_64` architecture.
 
+use super::PAGE_SIZE;
```

Comment:
move it near to line 10? They both imports from the crate itself.

Reply:
changed.